packages-next/types/src/config/access-control.ts

```diff
@@ -5,50 +5,40 @@ type BaseAccessArgs = {
   session: any;
   listKey: string;
   context: KeystoneContext;
```

Comment:
`gqlName` is also referenced in `access-control.mdx` and should be removed there.